ext/node/polyfills/internal/fs/handle.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import { EventEmitter } from "ext:deno_node/events.ts";
+import { Buffer } from "ext:deno_node/buffer.ts";
+import { promises } from "ext:deno_node/fs.ts";
+import {
+  BinaryOptionsArgument,
+  FileOptionsArgument,
+  TextOptionsArgument,
+} from "ext:deno_node/_fs/_fs_common.ts";
+
+export class FileHandle extends EventEmitter {
+  readonly rid: number;
```

Comment:
I think this should be `#rid` so it isn't a public field.3

Reply:
I agree, since it's already exposed as `fd` getter, let's make it `#rid` so it's not leaking to user land.